package.json

```diff
@@ -24,7 +24,7 @@
     "touch": "0.0.3"
   },
   "devDependencies": {
-    "coffee-script": "^1.9.1"
+    "coffee-script": "1.9.1"
```

Comment:
I don't think we use file watching anywhere in this repo (at least not officially/in a script), so I considered upgrading the version a "nice to have".